test/e2e/secrets_test.go

```diff
@@ -68,7 +68,7 @@ func testUserData(t *testing.T) {
 	tc, err := NewTestContext()
 	require.NoError(t, err)
 
-	pubKey, err := tc.getExpectedPublicKey()
+	_, pubKey, err := tc.getExpectedKeyPair()
 	require.NoError(t, err, "error determining expected public key")
```

Comment:
the private key processing can fail too, leading to a  "error determining expected private key"
Consider:
"[error getting the expected public/private key pair](https://github.com/openshift/windows-machine-config-operator/pull/508/commits/6f17a39559bf0cfc1bcd5c6eb2d98d5960103d99#diff-f08d14d1ac64442e0171050a9b9b70b50cfc7e1f0bc0eab1badc1e4a827252e3R273)"